code/game/machinery/PDApainter.dm

```diff
@@ -267,3 +261,155 @@
 /obj/machinery/pdapainter/power_change()
 	..()
 	update_icon()
+
+
+/proc/get_cardstyle_by_jobname(jobname)
```

Comment:
I feel like both `get_cardstyle_by_jobname` and `get_hud_by_jobname` could both be assimilated into one, there's a lot of repetition, or just handled in `var/valid_jobs` like it was previously

Reply:
likely no.
```
(card style)
			"Botanist" = "serv",
			"Bartender" = "serv",
			"Cook" = "serv",
```
```
(hud style)
			"Bartender" = "bartender",
			"Cook" = "cook",
			"Botanist" = "botanist",
```
each of them has different values and they can't be put together. but also, syncing their names isn't a good idea, and putting them into a single proc will make this more uglier as well,

Reply:
I forgot to put another reason too.
`get_hud_by_jobname`
`get_cardstyle_by_jobname`
proc names are obvious to realise what they do.